.expeditor/publish-release-notes.sh

```diff
@@ -20,7 +20,7 @@ pushd ./automate.wiki
   cat >./Current-Release-Notes.md <<EOH
 ## Upgrade Impact
 
-If you are upgrading from a version prior to 20190410001346, please read our [Important Compliance Outage Announcement](https://discourse.chef.io/t/important-compliance-outage-information-on-automate-2-april-15th-upgrade/14909).
+ICYMI: If you are upgrading from a version prior to 20190410001346, please read our [Important Compliance Outage Announcement](https://discourse.chef.io/t/important-compliance-outage-information-on-automate-2-april-15th-upgrade/14909).
```

Comment:
Question - Do we want to modify the label for the link? I don't necessarily find "Important Compliance Outage Announcement" as an informative label. I've replaced that in the past with "importance announcement." Thoughts @sdelano?

Reply:
@susanev reviewed the original PR and we thought it was a helpful addition to provide context ([link](https://github.com/chef/automate/pull/169#discussion_r278343571))

Reply:
Makes sense! LGTM.

Reply:
The link text needs to describe where we are sending them out of the context of the sentence. im totally good with alternative wording but am worried that _important annoucement_ doesn't explain what the important announcement is about if we remove it from the context of the sentence. https://accessibility.digital.gov/content-design/links/

Reply:
I'm fine with keeping it as is then, @susanev. The explanation helped a lot!